rails-buildpack/2.7.5/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM ruby:2.7.5
+
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends \
+      file \
+      git \
+      unzip \
+      curl \
+      autoconf \
+      automake \
+      default-libmysqlclient-dev \
+      default-mysql-client \
+      g++ \
+      gcc \
+      gnupg \
+      patch \
+      make \
+      libbz2-dev \
+      libc6-dev \
+      liblzma-dev \
+      libmagickcore-dev \
+      libmagickwand-dev \
+      libreadline-dev \
+      libtool \
+      libxslt-dev \
+      libpq-dev \
+      libsqlite3-dev \
+      libxml2-dev \
+      qtbase5-dev qtchooser qt5-qmake qtbase5-dev-tools \
+      libqt5webkit5-dev \
+      gstreamer1.0-plugins-base \
+      gstreamer1.0-tools \
+      gstreamer1.0-x \
+      imagemagick \
+ && rm -rf /var/lib/apt/lists/*
+
+# Install Node JS 16.x
+RUN curl -sL https://deb.nodesource.com/setup_lts.x | bash - \
+   && apt-get install -y nodejs \
+   && rm -rf /var/lib/apt/lists/*
+RUN npm install -g yarn
+
+ENV LANG C.UTF-8
+ENV LC_ALL C.UTF-8i
```

Comment:
@zuccamia Out of curiosity, is there a reason you chose to go with UTF-8i? The Dockerfiles in other versions of rails-buildpack seem to be going with UTF-8, so I was just wondering!

Reply:
Whoa that's really good catch! Thanks @yunshine! It was a typo 😔 I fixed it in dd5bd74